analysers/analyser_merge_fuel_IT.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, config, logger = None):
                     mapping1 = {
                         'ref:mise': 'idImpianto',
                         'operator': lambda res: italian_strings.normalize_common(res['Gestore']),
-                        'brand': 'Bandiera',
+                        'brand': lambda res: res['Bandiera'] if res['Bandiera'] != 'Pompe Bianche' else Mapping.delete_tag,
```

Comment:
Since you exclude the data at load, you does not need this change.

Reply:
I need it in case of rebrand

Reply:
No. `res` is only OpenData. In this code you does not have access to OSM data.

Reply:
I've just tested it and it adds the row to remove the brand in the XML as expected for pompe bianche.. what do you mean?

Reply:
At the load step you exclude row with 'Pompe Bianche'. So in the folowing step on data process you have no more this data, and so the `if res['Bandiera'] != 'Pompe Bianche' ` is useless.

It you really want to suggest delete of OSM tags brand=Pompe Bianche, keep the Mapping.delete_tag, but you have to include the the row with ''Pompe Bianche' on load step.

Reply:
Check row 61

Reply:
Ho, sorry, I read the diff in opposite way!
So, it s good for me.

Reply:
Nevermind and thanks